src/utils.js

```diff
@@ -0,0 +1,48 @@
+const { chunk } = require('lodash')
+const { findSavedTripByDates } = require('./queries')
+const { differenceWith } = require('lodash')
+
+function canSaveNextTripsChunk(startExecTime, timeout) {
+  const executionTimeSeconds = (new Date() - startExecTime) / 1000
+  return executionTimeSeconds < timeout
+}
+
+async function restartKonnector(client) {
+  const args = {
+    slug: 'tracemob'
+  }
+  const jobCollection = client.collection('io.cozy.jobs')
+  return jobCollection.create('konnector', args)
+}
+
+function createChunks(tripsMetadata, chunkSize) {
+  return chunk(tripsMetadata, chunkSize)
+}
+
+async function keepOnlyNewTrips(trips, accountId) {
```

Comment:
We should pass (newTrips, oldTrips). This method should not do the request by itself

Reply:
Fair enough. As this is "just" a style issue and require some refacto, I'll leave it for a next PR to be able to test this PR quickly.